packages/xo-web/src/common/intl/locales/es.js

```diff
@@ -1353,7 +1353,7 @@ export default {
   addHostErrorTitle: undefined,
 
   // Original text: 'Host patches could not be homogenized.'
```

Comment:
The comment should probably be updated as well.

Reply:
We didn't change message.